frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -643,10 +637,9 @@ public static function apiCreate(\OmegaUp\Request $r): array {
             fn (string $alias) => \OmegaUp\Validators::alias($alias)
         );
         \OmegaUp\Validators::validateStringNonEmpty($r['contents'], 'contents');
-        /**
-         * @var null|array{tags?: mixed, before_ac?: mixed, difficulty?: mixed, quality?: mixed, statements?: mixed, source?: mixed, reason?: mixed, original?: mixed} $contents
-         */
-        $contents = json_decode($r['contents'], associative: true);
+        $contents = \OmegaUp\DAO\QualityNominations::getContents(
+            $r['contents']
+        );
```

Comment:
```suggestion
        $rawContents = $r->ensureString(
            'contents',
            fn (string $s) => \OmegaUp\Validators::stringNonEmpty($s),
        );

        $contents = \OmegaUp\DAO\QualityNominations::getContents($rawContents);
```

for consistency with the idiom used below.